gambling.txt

```diff
@@ -13,6 +13,14 @@
 #
 # Gambling list
 # ------------------------------------[FILTERS]-------------------------------------
+0.0.0.0 promo.casinoroom.com
+0.0.0.0 kaszinoonline.net
+0.0.0.0 icecasino.com
+0.0.0.0 fezbet.com
```

Comment:
can you pull a screendump on this one for prove

![image](https://user-images.githubusercontent.com/44526987/184009406-6d85375d-ae3e-4dc2-ada4-a65b42ea0c0c.png)